include/deal.II/base/utilities.h

```diff
@@ -678,6 +678,56 @@ namespace Utilities
   }
 }
 
+// -------------------- Utility classes --------------------------
+namespace Utilities
+{
+  /**
+   * Some functions take as parameters references to vectors, and
+   * treat empty vectors as a sign that no work needs to be done
+   * (for example, compute(() in TensorProductPolynomials. This
```

Comment:
too many parentheses. If you write it as `TensorProductPolynomials::compute()`, then doxygen will automatically link to it.